src/AudioIOBase.cpp

```diff
@@ -197,7 +197,8 @@ void AudioIOBase::HandleDeviceChange()
       captureParameters.suggestedLatency =
          Pa_GetDeviceInfo(recDeviceNum)->defaultLowInputLatency;
    else
-      captureParameters.suggestedLatency = DEFAULT_LATENCY_CORRECTION/1000.0;
+      captureParameters.suggestedLatency =
+         AudioIOLatencyCorrection.GetDefault()/1000.0;
```

Comment:
We really should think about (gradually) switching to std::chrono at some point.

Reply:
Yes.  Maybe than a Settings<std::chrono::milliseconds> ?